test/fixtures.js

```diff
@@ -0,0 +1,110 @@
+const {api} = require('./api')
+
+const registryBytecode = require('./bytecodes/registry')
+const registryAbi = require('../src/contracts/abi/SimpleRegistry.json')
+
+let account = null
+let registry = null
+
+function deploy (abi, bytecode, parameters) {
```

Comment:
👍 

As we discussed yesterday, we may want to move them out into a separate repo/project/whatever to make them easily accessible for testing other code.